lib/browser/rpc-server.js

```diff
@@ -146,6 +146,30 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = function (meta, args, callIntoRenderer) {
```

Comment:
Since this is a new method, feel free to make this a `=>` function if you'd like.